web/src/ImpactDetailMode.svelte

```diff
@@ -10,11 +10,19 @@
   export let road: Feature;
 
   // TODO Weird to modify the input like this?
-  road.properties!.kind = "focus";
+  let props = road.properties!;
+  props.kind = "focus";
 
-  let routes = $backend!.getImpactsOnRoad(road.properties!.id);
+  let routes = $backend!.getImpactsOnRoad(props.id);
   let idx = 0;
 
+  if (routes.length == 0) {
+    window.alert(
```

Comment:
Why is this happening? The count of trips crossing a road before and after edits is changing, but we can't produce any specific examples?
https://github.com/a-b-street/ltn/blob/d1e32b0846c7b9fbf352bdab60fc726f4bc3b345/backend/src/impact.rs#L93
Right now we're insisting both the path before and after edits exists. When one of them doesn't exist, the count will be lower, but we don't return it yet.

I think it's probably better to show one of the paths is missing, similar to #181. I am not sure what's going on yet, but I suspect the route happens to start or end right on a road with a filter, and so it outright gives up instead of going the other direction on the road. So both parts of #181 may actually be pretty important to get better results here!